config.yml

```diff
@@ -28,10 +28,10 @@ detection_pipeline:
   defisheye_params:
     camera_matrix: [[2028, 0, 1954.1], [0, 2029.6, 1055.1], [ 0, 0, 1]]
     distortion_params: [[-0.24083, 0.10647, 0.00083113, 0.0001802, -0.025874]]
-    input_image_size: [3840, 2160]
-  target_classes: [2]
+    input_image_size: [1280, 720]
```

Comment:
Should remain `[3840, 2160]`, it's the size at which the distortion matrix was computed.